module-with-no-tests/src/main/java/org/apache/flink/connector/testing/SomeClass2.java

```diff
@@ -19,4 +19,7 @@
 package org.apache.flink.connector.testing;
 
 /** A dummy class; only exists so that the compile/packaging plugins have something to do. */
-public class SomeClass {}
+public class SomeClass2 {
```

Comment:
Why is this class being modified?

Reply:
not modified: the original class is still named SomeClass and is located in module-with-tests module (the former test-project). This SomeClass2 is another one for the module-with-no-tests module.

Reply:
no more need for this SomeClass**2** name as the new archunit rule is on the name of the class, so the class in module-with-tests is now called SomeConnectorClass.